src/source/reference/get-metrics-report.rst

```diff
@@ -4,6 +4,24 @@
 GET /metrics/report
 =======================
 
+V2
+~~~
+
+.. list-table::
+ :widths: 30 70
+
+ * - Scope
+   - ``console_ro`` or ``api_ro``
+
+ * - Accept
+   - ``application/sellside.metrics.ad-hit-v2+json`` or ``application/sellside.metrics.ad-session-v2+json``
+
+
+V2 replaces market specific currency with our internal universal micros currency. 
```

Comment:
There is no market-specific currency in this endpoint. It's only cents from what I remember.
Can we have a small seciton on bidding and micros that all these new endpoints can link towards?
with the explanation of why it was introduced in the first place

Reply:
We have $ in kjca and Euros in the rest of the markets